export type SectionOverride = {
  components: string;
  procedure: string;
  specifications: string;
  troubleshooting: string;
};

export const seedSectionOverrides: Record<string, SectionOverride> = {
  "etbr-destroyer-bag": {
    "components": "APEBD 001\nALPHAGEN EtBr Destroyer Bag: 20 pack\nAPEBD 001-1\nALPHAGEN EtBr Destroyer Bag: 10 pack\nAPEBD 003\nALPHAGEN EtBr Destroyer Bag & Sprayer Starter Kit: 5 bags, 1 x 200ml bottle, & 1 sprayer Head",
    "procedure": "Ethidium Bromide\nEthidium Bromide containing buffer :\nStep 1 : Tear open the package and take out the ALPHAGEN EtBr Destroyer Bag. Use the product immediately or seal it into a ziplock bag to avoid prolonged exposure to the air.\nStep 2 : Put the EtBr Destroyer into the EtBr-containing buffer. Completely submerge the bag so that the entire Destroyer bag is in the solution.\nStep 3 : Destroyer Working Timetable\nStep 4 : When the treated buffer becomes clear, please use a UV light (254nm) to observe the EtBr-content remaining. If exposure to the UV light displays no fluorescence and the O.D. value is equal to that of d.d.H2O it means that the EtBr containing buffer has been totally destroyed. Step 5 The ALPHAGEN EtBr Destroyer Bag has absorbed and destroyed all of the EtBr content in the liquid. Following treatment the bag can be thrown away as a non-hazardous item in compliance with the local waste removal regulations.\nEthidium Bromide containing gels :\nStep 1 : Tear open the package and take out the ALPHAGEN EtBr Destroyer Bag onto the container which fills with 3L d.d.H2O\nStep 2 : 30 minutes later, you can drop the EtBr-contaminated gels into this container filling with the solution of ALPHAGEN EtBr Destroyer Bag.\nStep 3 : You can drop as many as gels into the container during the 6 months period, but the total amount of EtBr in all gels can&rsquo;t be over 30mg and the solution of ALPHAGEN EtBr Destroyer Bag has to cover the whole gels. The last gel dropped into the container only can be thrown away after 24 hours treatment.\nStep4 : When the treated gel becomes clear, please use the UV light box (UV Wavelength 254nm) to ensure the Ethidium Bromide has been totally destroyed.\nStep 5 : After 6 months, no matter how much EtBr has been destroyed, the solution of ALPHAGEN EtBr Destroyer Bag can&rsquo;t be used any more. The users need to clean up the container for another new solution.\nStep 6 : After gels containing EtBr which have been destroyed can be collected as the non-hazardous item in compliance with the local waste removal regulations.\nSYBR Green I\nBuffer containing SYBR Green I :\nUse the same procedure as for the Ethidium Bromide containing buffer. However please note that 1.) The decontamination time is different to that of Ethidium Bromide containing buffer, and 2.) You must use UV light box (wavelength 488nm) to observe SYBR Green I.\nGels containing SYBR Green I:\nThe procedure is the same as that of the Ethidium Bromide containing gels. However please note that 1.) The decontamination time is different to that of Ethidium Bromide containing gels, and 2.) You must use UV light box (wavelength 488nm) to observe SYBR Green I.\nSYBR Gold\nBuffer containing SYBR Gold :\nUse the same procedure as for the Ethidium Bromide containing buffer. However please note that 1.) The decontamination time is different to that of Ethidium Bromide containing buffer, and 2.) You must use UV light box (wavelength 488nm) to observe SYBR Gold.\nGels containing SYBR Gold :\nThe procedures are the same as the usage for Ethidium Bromide containing gels. However please note that 1.) The decontamination time is different to that of Ethidium Bromide containing gels, and 2.) Please use the UV light box (wavelength 488nm) to observe the SYBR Gold.\nSYPRO Orange\nBuffer containing SYPROOrange :\nUse the same procedure as for the Ethidium Bromide containing buffer. However please note that 1.) The decontamination time is different to that of Ethidium Bromide containing buffer, and 2.) You must use UV light box (wavelength 488nm) to observe SYPRO Orange.\nGels containing SYPRO Orange :\nThe procedure is the same as that of the Ethidium Bromide containing gels. However please note that 1.) The decontamination time is different to that of Ethidium Bromide containing gels, 2.) Please use the UV light box (wavelength 488nm) to observe the SYPRO Orange.",
    "specifications": "ALPHAGEN EtBr Destroyer is provided in two different formats for the treatment of both solid and liquid Ethidium Bromide contaminant.\nThe ready pack ALPHAGEN EtBr Destroyer Bag is idea for the treatment of liquid Ethidium Bromide contaminant while Sprayer can be used for the treatment of solid contaminant waste including electrophoresis gels, glassware, paper towels, gloves, laboratory equipment, bench surface etc.\nEffective: ALPHAGEN EtBr Destroyer is a laboratory reagent intended for the removal and destruction of Ethidium Bromide contamination. This effect can be monitored and confirmed by UV light exposure whereby once the EtBr has been destroyed the fluorescence will disappear.\nFast: For general protection of uncontaminated area, spray the EtBr Destroyer on the entire working area, leave for about 5 minutes, then wipe it dry with paper towel. When dealing with more seriously contaminated liquid samples the EtBr Destroyer bag is an ideal solution. Three litters of double distilled H2O containing 30mg of Ethidium Bromide can be destroyed in a matter of days using ALPHAGEN easy-to-follow up protocol with the bag format.\nSpecific: Two specific designs are for different purposes : Ethidium Bromide Destroyer bags and Ethidium Bromide Destroyer Spray\nEconomical: ALPHAGEN EtBr Destroyer Sprayer bottles can be refilled after use and the bags are available in both a 10 and 20 pack. In comparison to the costs associated with properly disposing of hazardous materials the EtBr Destroyer products save your money in the long run. You can even re-use your buffer after treatment! One of the big economical benefits of using this product is that after using the ALPHAGEN EtBr Destroyer, the oncecontaminated buffer can be filtrated and use again without worry. 5. Safe: ALPHAGEN EtBr Destroyer has been demonstrated by Ames test to safely and effectively destroy EtBr without any mutagenic effect. The AMES test result data supports that the genotoxicity of EtBr contaminated solution, treated by both the ALPHAGEN EtBr Destroyer Band and Sprayer was reduced significantly and a negative result was determined.",
    "troubleshooting": ""
  },
  "etbr-destroyer-sprayer": {
    "components": "APEBD002\nEtBr Destroyer Sprayer Pack:\n2 x 200ml bottles and 1 sprayer head\nAPEBD002-1\nEtBr Destroyer Supplement: 2\nX 200ml refill bottles, no sprayer head\nAPEBD002-2\nEtBr Destroyer Supplement: 1\nx 1L refill bottle, no sprayer head\nAPEBD 003\nAlphagen EtBr Destroyer-Bag & Sprayer",
    "procedure": "Ethidium Bromide\nStep 1 : Open the bottle cap of the sprayer and insert the sprayer head. The sprayer head has to be screwed onto the bottle tightly.\nStep 2 : Spray onto the contaminated target area. The liquid has to completely cover the entire contaminant.\nStep 3 : Spray onto the bench for a) general protection of potential contaminated area. Spray the whole working area and wipe down after 5 minutes b) For known contaminated area. Spray on the specific working area on the following working time (see protocol)\nStep 4 : Please use the portable UV light box (UV Wavelength 254nm) to ensure the Ethidium Bromide has been totally destroyed.\nSYBR Green I\nSpray on bench : Use the same procedure as for the Ethidium Bromide working area. However please note that 1.) The decontamination time is different to that of EtBr containing gels and 2.) You must use UV light box (wavelength 488nm) to observe SYBR Green.\nSpray on gels stained with SYBR Green : The procedure is the same as that of the Ethidium Bromide containing gels. However please note that 1.) The decontamination time is different to that of EtBr containing gels . For example, if you spray 0.9ml (about 3 sprays) on the gel stained with SYBR Green (concentration 0.5~1ug/ml), then after 1 hour, the SYBR Green will have been destroyed. Please use the UV light box (wavelength 488nm) to observe the SYBR Green.\nSYBR Gold\nSpray on bench : Use the same procedure as for the Ethidium Bromide working area. However please note that 1.) The decontamination time is different to that of EtBr containing gels and 2.) You must use UV light box (wavelength 488nm) to observe SYBR Gold.\nSpray on bench : Use the same procedure as for the Ethidium Bromide working area. However please note that 1.) The decontamination time is different to that of EtBr containing gels and 2.) You must use UV light box (wavelength 488nm) to observe SYBR Gold. Spray on gels stained with SYBR Gold : The procedures are the same as the usage for Ethidium Bromide containing gels. However please note that 1.) The decontamination time is different to that of EtBr containing gels. For example, if you spray 0.9ml (about 3 sprays) on the gel stained with SYBR Gold (concentration 0.5~1ug/ml), then after 1 hour, the SYBR Gold will have been destroyed. Please use the UV light box (wavelength 488nm) to observe the SYBR Gold.\nSYPRO Orange\nSpray on bench : Use the same procedure as for the Ethidium Bromide working area. However please note that 1.) The decontamination time is different to that of EtBr containing gels and 2.) You must use UV light box (wavelength 488nm) to observe SYBR Orange.\nSpray on gels stained with SYPRO Orange : The procedure is the same as that of the Ethidium Bromide containing gels. However please note that 1.) The decontamination time is different to that of EtBr containing gels. For example, if you spray 5.4ml (about 18 sprays) on the gel stained with SYPRO Orange (concentration 0.5~1ug/ml), then after 3 hours, the SYPRO Orange will have been destroyed. Please use the UV light box (wavelength 488nm) to observe the SYPRO Orange.",
    "specifications": "ALPHAGEN EtBr Destroyer is provided in two different formats for the treatment of both solid and liquid Ethidium Bromide contaminant.\nThe ready pack ALPHAGEN EtBr Destroyer Bag is idea for the treatment of liquid Ethidium Bromide contaminant while Sprayer can be used for the treatment of solid contaminant waste including electrophoresis gels, glassware, paper towels, gloves, laboratory equipment, bench surface etc.\nEffective: ALPHAGEN EtBr Destroyer is a laboratory reagent intended for the removal and destruction of Ethidium Bromide contamination. This effect can be monitored and confirmed by UV light exposure whereby once the EtBr has been destroyed the fluorescence will disappear.\nFast: For general protection of uncontaminated area, spray the EtBr Destroyer on the entire working area, leave for about 5 minutes, then wipe it dry with paper towel. When dealing with more seriously contaminated liquid samples the EtBr Destroyer bag is an ideal solution. Three litters of double distilled H2O containing 30mg of Ethidium Bromide can be destroyed in a matter of days using ALPHAGEN easy-to-follow up protocol with the bag format.\nSpecific: Two specific designs are for different purposes : Ethidium Bromide Destroyer bags and Ethidium Bromide Destroyer Spray\nEconomical: ALPHAGEN EtBr Destroyer Sprayer bottles can be refilled after use and the bags are available in both a 10 and 20 pack. In comparison to the costs associated with properly disposing of hazardous materials the EtBr Destroyer products save your money in the long run. You can even re-use your buffer after treatment! One of the big economical benefits of using this product is that after using the ALPHAGEN EtBr Destroyer, the oncecontaminated buffer can be filtrated and use again without worry. 5. Safe: ALPHAGEN EtBr Destroyer has been demonstrated by Ames test to safely and effectively destroy EtBr without any mutagenic effect. The AMES test result data supports that the genotoxicity of EtBr contaminated solution, treated by both the ALPHAGEN EtBr Destroyer Band and Sprayer was reduced significantly and a negative result was determined.",
    "troubleshooting": ""
  },
  "pcr-cleanup-mini-kit": {
    "components": "APPCK 004\n(4 preps)\nAPPCK 050\n(50 preps)\nAPPCK 200\n(200 preps)\nPC Buffer\n1.5 ml x 2\n30 ml\n110 ml\nWash Buffer (concentrate)\n1 ml\n12.5 ml\n45 ml\nElution Buffer\n0.5 ml\n5 ml\n20 ml\nPC Mini Column\n4 pcs\n50 pcs\n200 pcs\nCollection Tube\n4 pcs\n50 pcs\n200 pcs\nElution Tube\n4 pcs\n50 pcs\n200 pcs\nUser Manual\n1\n1\n1",
    "procedure": "",
    "specifications": "※ Principle : spin column (silica matrix)\n※ DNA Binding capacity of spin column : 20 ul\n※ Sample size : up to 100 ul of reaction solution\n※ DNA size : 65 bp ~ 10 kbp\n※ Recovery : 90% ~ 95% for PCR clean-up\n※ Operation time : 10 ~ 15 min\n※ E lution volume : 40 ul\nImportant Notes :\n※ Buffer provided in this kit contain irritants. Wear gloves and lab coat when handling these buffer.\n※ Add the required volume of ethanol (96~100%) to Wash Buffer before use.\n※ Centrifugation steps are done by a microcentrifuge capable of the speed at 11,000 ~1,8000 x g.",
    "troubleshooting": ""
  },
  "gel-pcr-purification-kit": {
    "components": "APGCK 004\nAPGCK 050\nAPGCK 100\nAPGCK 300\nDF Buffer\n1.5 ml x 2\n40 ml\n80 ml\n240 ml\nWash Buffer (concentrate)\n1.0 ml\n15 ml\n25 ml\n50 ml\nElution Buffer\n0.5 ml\n5 ml\n6 ml\n30 ml\nDF Column\n4 pcs\n50 pcs\n100 pcs\n300 pcs\nCollection Tube\n4 pcs\n50 pcs\n100 pcs\n300 pcs\nUser Manual\n1\n1\n1\n1",
    "procedure": "",
    "specifications": "Specification:\nPrinciple : spin column (silica matrix)\nDNA Binding capacity of spin column : 20 ug\nSample size : up to 300 mg of agarose gel . up to 300 mg of agarose gel\nDNA size : 65 bp ~ 10 kbp\nRecovery : 70% ~ 85% for Gel extraction . 90% ~ 95% for PCR clean-up\nBinding capacity : 10 ~ 20 min\nOperation time : 40 ul\nImportant Notes:\nBuffer provided in this kit contain irritants. Wear gloves and lab coat when handling these buffer.\nAdd the required volume of ethanol (96~100%) to Wash Buffer before use.\nCentrifugation steps are done by a microcentrifuge capable of the speed at 11,000 ~1,8000 x g.",
    "troubleshooting": "For Gel Extraction\nProblems\nPossible reasons\nSolutions\nThe gel slice is hard to dissolve\nAgarose gel of high percentage (&gt; 2 %) is used\nAdd 1000 ul of DF Buffer to 1 volume of the gel slice.\nThe size of the gel slice is too large\nIf the gel slice is more than 300 mg, separate it into multiple tubes.\nLow or none recovery of DNA fragment\nThe column is loaded with too much agarose gel\nThe maximum volume of the gel slice is 300 mg per column.\nElution of DNA fragment is not efficient\nMake sure the pH of Elution Buffer or ddH2O is between 7.0- 8.5.\nMake sure that the elution solution has been completely absorbed by the membrane before centrifuge.\nThe size of DNA fragment is larger than 5 Kb\nPreheat the elution solution to 60 °C before use.\nEluted DNA contains non-specific\nDNA fragment\nContaminated scalpel\nUsing a new or clean scalpel.\nDNA fragment is denatured.\nIncubate eluted DNA at 95 °C for 2 min, then cool down slowly to reanneal denatured DNA.\nPoor perfor- mance in the downstream applications\nSalt residue remains in eluted DNA fragment\nWash the column twice with Wash Buffer.\nEthanol residue remains in eluted DNA fragment\nDo discard the flow-through after washing with Wash Buffer and centrifuge for an additional 3 min.\nFor PCR Clean-Up\nProblems\nPossible reasons\nSolutions\nLow or none recovery of DNA fragment\nApply more than 100 ul of PCR product\nIf the PCR product is more than 100 ul, separate it into multiple tubes.\nElution of DNA fragment is not efficient\nMake sure the pH of Elution Buffer or ddH2O is between 7.0- 8.5.\nMake sure that the elution solution has been completely absorbed by the column membrane before centrifuga- tion.\nThe size of DNA fragment is larger than 5 Kb\nPreheat the elution solution to 60 °C before use.\nPoor perfor- mance in the downstream applications\nSalt residue remains in eluted DNA\nWash the column twice with Wash Buffer.\nEthanol residue remains in eluted DNA\nDo discard the flow-through after washing with Wash Buffer and centrifuge for an additional 3 min."
  },
  "genomic-dna-cleanup-kit": {
    "components": "APGDC 004\n(4 preps)\nAPGDC 050\n(50 preps)\nAPGDC 200\n(200 preps)\nGC Buffer\n1.5 ml x 2\n30 ml\n120 ml\nWash Buffer (concentrate)\n1 ml\n10 ml\n40 ml\nElution Buffer\n1.5 ml\n15 ml\n50 ml\nGC Mini Column\n4 pcs\n50 pcs\n200 pcs\nCollection Tube\n4 pcs\n50 pcs\n200 pcs\nElution Tube\n4 pcs\n50 pcs\n200 pcs\nUser Manual\n1\n1\n1",
    "procedure": "",
    "specifications": "※ Principle : spin column (silica matrix)\n※ DNA Binding capacity of spin column : 50 ul .\n※ Sample size : up to 100 ul of reaction solution.\n※ Recovery : 80% ~ 95%\n※ Operation time : 10 ~ 15 min\n※ Elution volume : 50 ul\nImportant Notes :\n※ Buffer provided in this kit contain irritants. Wear gloves and lab coat when handling these buffer.\n※ Add the required volume of ethanol (96~100%) to Wash Buffer before use.\n※ Centrifugation steps are done by a microcentrifuge capable of the speed at 10,000 x g.\n※ Heat the Elution Buffer to 65 °C before use.",
    "troubleshooting": ""
  },
  "blood-genomic-dna-extraction": {
    "components": "APBGK 004\n(4 preps)\nAPBGK 050\n(50 preps)\nAPBGK 100\n(100 preps)\nAPBGK 200\n(200 preps)\nBG Buffer\n1.5 ml\n15 ml\n30 ml\n60 ml\nGW Buffer (concentrate)\n1.3 ml\n22 ml\n44 ml\n88 ml\nWash Buffer (concentrate)\n1 ml\n10 ml\n20 ml\n40 ml\nElution Buffer\n1 ml\n15 ml\n30 ml\n60 ml\nProteinase K\n1 mg\n11 mg\n11 mg x 2\n11 mg x 4\nBG Mini Column\n4 pcs\n50 pcs\n100 pcs\n200 pcs\nCollection Tube\n8 pcs\n100 pcs\n200 pcs\n400 pcs\nElution Tube\n4 pcs\n50 pcs\n100 pcs\n200 pcs\nUser Manual\n1\n1\n1\n1",
    "procedure": "",
    "specifications": "※ Principle : mini spin column (silica matrix)\n※ Sample size : 1. Up to 200 ul whole blood, serum, plasma, body fluids.\n2. Up to 5 x 10 6 cultured cells.\n※ Operation time : 30 minutes\n※ Binding capacity : up to 60 ug total DNA / column\n※ Expected yield : 4 ~ 8 ug / prep\n※ Column applicability : centrifugation and vaccum\n※ Minimum elution volume : 50 ul\nImportant Notes :\n※ Buffers provided in this system contain irritants. Wear gloves and lab coat when\nhandling these buffers.\n※ Add 1.1 ml sterile ddH 2 O to Proteinase K tube to make a 10 mg/ml stock solution.\nMake sure that Proteinase K has been completely dissolved. Stor e the stock\nsolution at 4 ℃ .\n※ Add ethanol (96- 100 %) to GW Buffer and Wash Buffer when first open.\n※ Prepare dry baths or water baths before the operation. One to 60℃ for step 2. Lysis.",
    "troubleshooting": ""
  },
  "blood-cultured-cell-dna-extraction": {
    "components": "APBGR 004\n(4 preps)\nAPBGR 100\n(100 preps)\nAPBGR 200\n(200 preps)\nRBC Lysis Buffer\n7 ml\n135 ml\n270 ml\nTG Buffer\n1.5 ml\n30 ml\n60 ml\nBG Buffer\n1.5 ml\n40 ml\n80 ml\nBW Buffer\n1.5 ml x 2\n45 ml\n90 ml\nWash Buffer (concentrate)\n1 ml\n25 ml\n35 ml\nElution Buffer\n1.5 ml\n30 ml\n60 ml\nBG Mini Column\n4 pcs\n100 pcs\n200 pcs\nCollection Tube\n8 pcs\n200 pcs\n400 pcs\nUser Manual\n1\n1\n1",
    "procedure": "",
    "specifications": "※ Principle : mini spin column (silica matrix)\n※ Sample size : 1. Up to 300 ul of Whole blood\n2. Up to 200 ul of frozen blood\n3. Up to 200 ul of buffy coat\n4. Up to 1 x 10 7 of Cultured animal cells\n5. Up to 1 x 10 9 of Cultured bacterial cells\n※ Operation time : 30 minutes\n※ Binding capacity : up to 60 ug total DNA / column\n※ Expected yield : 4 ~ 8 ug / prep\n※ Column applicability : centrifugation and vaccum\n※ Minimum elution volume : 50 ul\nImportant Notes :\n※ Buffers provided in this system contain irritants. Wear gloves and lab coat when handling these buffers.\n※ Add ethanol (96- 100 %) to Wash Buffer when first open.",
    "troubleshooting": "Problems\nPossible reasons\nSolutions\nLow or no yield of genomic DNA\nIncorrect preparation of Wash Buffer.\nWash Buffer is not mixed with ethanol before use.\nMake sure that the correct volumes of ethanol (96 - 100 %) is added into the Wash Buffer when first open ed.\nThe volume or the percentage of ethanol is not correct before adding it into the Wash Buffer.\nMake sure that the correct volumes of ethanol (96 - 100 %) is added into the Wash Buffer when first used.\nElution of genomic DNA is not efficient.\nThe Elution Buffer is not completely absorbed by the column membrane.\nAfter the Elution Buffer is added, stand the BG Column for 2 min before centrifugation.\nThe pH value of Elution Buffer is not correct.\nMake sure the pH value of Elution Buffer is between 7.5- 8.5.\nColumn is clogged\nSample is too viscous.\nReduce the sample volume.\nDegradation of eluted DNA\nSample is old.\nAlways use fresh or well-stored sample viral nucleic acid extraction."
  },
  "plant-genomic-dna-extraction": {
    "components": "APPGK 004\n(4 preps)\nAPPGK 050\n(50 preps)\nAPPGK 100\n(100 preps)\nPG1 Buffer\n2 ml\n25 ml\n55 ml\nPG2 Buffer\n1 ml\n8 ml\n15 ml\nPG3 Buffer (concentrate)\n1.5 ml\n15 ml\n30 ml\nGW Buffer (concentrate)\n0.8 ml\n13 ml\n26 ml\nWash Buffer (concentrate)\n1.5 ml\n15 ml\n30 ml\nElution Buffer\n1.5 ml\n15 ml\n30 ml\nRNase A (lyophilized)\n1.5 mg\n22 mg\n43 mg\nFilter Column\n4 pcs\n50 pcs\n100 pcs\nPG Column\n4 pcs\n50 pcs\n100 pcs\nCollection Tube\n8 pcs\n100 pcs\n200 pcs\nUser Manual\n1\n1\n1",
    "procedure": "",
    "specifications": "※ Principle : mini spin column (silica matrix)\n※ Sample size : 1. Wet weight ≦ 100 mg\n2. Dry weight ≦ 20 mg\n※ Operation time : 30 ~ 60 minutes\n※ Binding capacity : up to 60 ug total DNA / column\n※ Expected yield : 5 ~ 40 ug /prep\n※ Column applicability : centrifugation and vaccum\n※ Minimum elution volume : 5 0 ul\nImportant Notes :\n※ Buffers provided in this system contain irritants. Wear gloves and lab coat when handling these buffers.\n※ Check PG1 Buffer before use, Warm PG1 Buffer at 60℃ for 5 min if any precipitate formd.\n※ Preheat dry baths or water baths to 65℃ before the operation.\n※ Add required ethanol (96-100%) to PG3 Buffer, GW and Wash before use.\n※ Store RNase A at -20℃ upon recipit of kit. Add sterile ddH 2 O to RNase A tube to make a 50 mg/ml stock solution. Vortex and make sure that RNase A has been completely dissolved. Store the stock solution at -20℃ .\n※ All centrifuge steps are done at full speed(~ 18,000 x g) in a microcentrifuge.",
    "troubleshooting": ""
  },
  "tissue-genomic-dna-extraction": {
    "components": "APTGK 004\n(4 preps)\nAPTGK 050\n(50 preps)\nAPTGK 100\n(100 preps)\nAPTGK 200\n(200 preps)\nTG1 Buffer\n1.5 ml\n15 ml\n30 ml\n60 ml\nTG2 Buffer\n1.5 ml\n15 ml\n30 ml\n60 ml\nProteinase K\n(lyophilized)\n1 mg\n11 mg\n11 mg x 2\n11 mg x 4\nGW Buffer\n(concentrate)\n1.3 ml\n22 ml\n44 ml\n88 ml\nWash Buffer\n(concentrate)\n1 ml\n10 ml\n20 ml\n40 ml\nElution Buffer\n1 ml\n15 ml\n30 ml\n60 ml\nTG Mini Column\n4 pcs\n50 pcs\n100 pcs\n200 pcs\nCollection Tube\n8 pcs\n100 pcs\n200 pcs\n400 pcs\nElution Tube\n4 pcs\n50 pcs\n100 pcs\n200 pcs\nUser Manual\n1\n1\n1\n1",
    "procedure": "",
    "specifications": "※ Principle : mini spin column (silica matrix)\n※ Sample size : 1. Up to 25 mg animal tissue\n2. 1.2 cm mouse tail\n3. Up to 10 7 cultured cells\n※ Operation time : 30 ~ 60 minutes\n※ Binding capacity : up to 60 ug total DNA / column\n※ Expected yield : 15 ~35 ug / prep\n※ Column applicability : centrifugation and vaccum\n※ Minimum elution volume : 50 ul\nImportant Notes :\n※ Buffers provided in this system contain irritants. Wear gloves and lab coat when handling these buffers.\n※ Add 1.1 ml sterile ddH2O to Proteinase K tube to make a 10 mg/ml stock solution. Make sure that Proteinase K has been completely dissolved. Store the stock solution at 4 ℃ .\n※ Add ethanol (96- 100 %) to GW Buffer and Wash Buffer when first open.\n※ Prepare dry baths or water baths before the operation. One to 60℃ for step 2.1 and the other to 70℃ for step 2.2.\n※ Preheat the Elution Buffer to 70℃ for step 7.\n※ All centrifuge steps are done at full speed(~ 18,000 x g) in a microcentrifuge.",
    "troubleshooting": ""
  },
  "high-yield-plasmid-extraction": {
    "components": "APHPD 004\nAPHPD 100\nAPHPD 300\nPD1 Buffer\n1.5 ml\n30 ml\n90 ml\nPD2 Buffer\n1.5 ml\n30 ml\n90 ml\nPD3 Buffer\n1.5 ml\n40 ml\n120 ml\nPDW Buffer (concentrate)\n1.3 ml\n35 ml\n98 ml\nWash Buffer (concentrate)\n1.0 ml\n20 ml\n50 ml\nElution Buffer\n0.5 ml\n15 ml\n35 ml\nPD Column\n4 pcs\n100 pcs\n300 pcs\nCollection Tube\n4 pcs\n100 pcs\n300 pcs\nRNase A (Lyophilized)\n0.15 mg\n3 mg\n9 mg\nUser Manual\n1\n1\n1",
    "procedure": "",
    "specifications": "※ Principle : mini spin column (silica matrix)\n※ Sample size : 1 ~ 5 ml\n※ Size of plasmid or construct : &lt; 15 kb\n※ Operation time : &lt; 25 minutes\n※ Typical Yield : 25 ~ 40 ug\n※ Binding capacity : 60 ug/ column\n※ Column applicability : centrifugation and vaccum\nImportant Notes:\n※ Store RNase A at -20 °C upon recipit of kit.\n※ Add 0.5 ml of PD1 Buffer to a RNase A tube, Dissolve the RNase A by vortexing. Briefly spin the tube and transfer the total RNase A mixture back to the PD1 bottle, mix well by vortexing and store the PD1 buffer at 4 °C.\n※ If precipitates have formed in PD2 Buffer, warm the buffer in 37°C waterbath to dissolve precipitates.\n※ Preparation of PDW Buffer and Wash Buffer by adding 96 ~100% ethanol (not provided) for first use.\n※ Centrifugation steps are done by a microcentrifuge capable of the speed at 11,000 ~1,8000 x g.",
    "troubleshooting": "Problems\nPossible reasons\nSolutions\nLow yield\nBacterial cells were not lysed completely\nToo many bacterial cells were used (OD600 &gt; 10). Separate the bacterial culture into multiple tubes.\nAfter PD3 Buffer addition, break up the precipitate by inverting to ensure higher yield.\nOvergrown of bacterial cells\nIncubation time should not be longer than 16 hours.\nBacterial cells were insufficient\nEnsure that bacterial cells have grown to an expected amount (OD600 &gt; 1) after incubation under suitable shaking modes.\nIncorrect DNA elution step\nEnsure that Elution Buffer was added and absorbed to the center of the PD Column matrix.\nIncomplete DNA Elution\nIf the size of DNA fragments is larger than 10 kb, use a preheated Elution Buffer (60~70°C) on the solution step to improve the elution efficiency.\nIncorrect preparation of PDW Buffer and Wash Buffer\nEnsure that the correct volume of ethanol (96 ~ 100 %) was added to PDW Buffer and Wash Buffer before use.\nEluted DNA does not perform well\nResidual ethanol contamination\nAfter Wash Step, dry the PD Column with an additional centrifugation at top speed (~18,000 x g) for 5 minutes or incubation at 60°C for 5 minutes.\nGenomic DNA Contamination\nLysate prepared improperly\nGently invert the tube after adding the PD2 Buffer. And the incubation time should not be longer than 5 minutes.\nDo Not use overgrown bacterial culture.\nRNA Contamination Plasmid DNA\nInsufficiency of RNase A activity in PD1 Buffer because of long-term storage\nPrior to using PD1 Buffer, ensure that RNase A was added. If RNase A added PD1 Buffer is out of date, add additional RNase A into PD1 Buffer to a concentration of 50 ug/ ml then store 4°C.\nToo many bacterial cells were used, reducing sample volume.\nSmearing or degrading of Plasmid DNA\nNuclease contamination\nIf used host cells have high nuclease activity (e.g., enA + strains), perform the following optional Wash Step to remove residuary nuclease.\na. After DNA Binding Step, add 400 ul of PDW Buffer into the PD Column and incubate for 2 minutes at room temperature.\nb. Centrifuge at full speed (~18,000 xg) for 30 seconds.\nc. Proceed to step 6.2.\nPlasmid DNA is not adequate for enzymatic digestions\nEluted plasmid DNA contains residual ethanol\nMake sure you have discarded the flow-through after washing with Wash Buffer (Step 6.2) and centrifuged for an additional 3 minutes (Step 7).\nDenatured Plasmid DNA migrate faster than supercoiled form during electrophoresis\nIncubation in PD2 Buffer too long\nDo not incubate the sample longer than 5 minute in PD2 Buffer"
  },
  "plasmid-extraction-mini-kit": {
    "components": "APPDE 004\nAPPDE 050\nAPPDE 100\nAPPDE 300\nPD1 Buffer\n1.0 ml\n13 ml\n25 ml\n75 ml\nPD2 Buffer\n1.0 ml\n13 ml\n25 ml\n75 ml\nPD3 Buffer\n1.5 ml\n18 ml\n35 ml\n105 ml\nPDW Buffer\n1.3 ml\n23 ml\n45 ml\n135 ml\nWash Buffer (concentrate)\n1.0 ml\n10 ml\n20 ml\n50 ml\nElution Buffer\n0.5 ml\n7 ml\n15 ml\n35 ml\nPD Column\n4 pcs\n50 pcs\n100 pcs\n300 pcs\nCollection Tube\n4 pcs\n50 pcs\n100 pcs\n300 pcs\nRNase A (Lyophilized)\n0.15 mg\n1.25 mg\n2.5 mg\n7.5 mg\nUser Manual\n1\n1\n1\n1",
    "procedure": "",
    "specifications": "※ Principle : mini spin column (silica matrix)\n※ Sample size : 1 ~ 3 ml\n※ Size of plasmid or construct : &lt; 15 kb\n※ Operation time : &lt; 25 minutes\n※ Typical Yield : 20 ~ 30 ug\n※ Binding capacity : 60 ug/ column\n※ Column applicability : centrifugation and vaccum\nImportant Notes:\n※ Store RNase A at -20 °C upon recipit of kit.\n※ Add 0.5 ml of PD1 Buffer to a RNase A tube, Dissolve the RNase A by vortexing. Briefly spin the tube and transfer the total RNase A mixture back to the PD1 bottle, mix well by vortexing and store the PD1 buffer at 4 °C.\n※ If precipitates have formed in PD2 Buffer , warm the buffer in 37°C waterbath to dissolve precipitates.\n※ Preparation of Wash Buffer by adding 96 ~100% ethanol (not provided) for first use.\n※ Centrifugation steps are done by a microcentrifuge capable of the speed at 11,000 ~1,8000 x g.",
    "troubleshooting": "Problems\nPossible reasons\nS olutions\nLow yield\nBacterial cells were not lysed completely\nToo many bacterial cells were used (OD600 &gt; 10). Separate the bacterial culture into multiple tubes.\nAfter PD3 Buffer addition, break up the precipitate by inverting to ensure higher yield.\nOvergrown of bacterial cells\nIncubation time should not be longer than 16 hours.\nBacterial cells were insufficient\nEnsure that bacterial cells have grown to an expected amount (OD600 &gt; 1) after incubation under suitable shaking modes.\nIncorrect DNA elution step\nEnsure that Elution Buffer was added and absorbed to the center of the PD Column matrix.\nIncomplete DNA Elution\nIf the size of DNA fragments is larger than 10 kb, use a preheated Elution Buffer (60~70°C) on the solution step to improve the elution efficiency.\nIncorrect preparation of Wash Buffer\nEnsure that the correct volume of ethanol (96 ~ 100 %) was added to Wash Buffer before use.\nEluted DNA does not perform well\nResidual ethanol contamination\nAfter Wash Step, dry the PD Column with an additional centrifugation at top speed (~18,000 x g) for 5 minutes or incubation at 60°C for 5 minutes.\nGenomic DNA Contamination\nLysate prepared improperly\nGently invert the tube after adding the PD2 Buffer. And the incubation time should not be longer than 5 minutes.\nDo Not use overgrown bacterial culture.\nRNA Contamination Plasmid DNA\nInsufficiency of RNase A activity in PD1 Buffer because of long-term storage\nPrior to using PD1 Buffer, ensure that RNase A was added. If RNase A added PD1 Buffer is out of date, add additional RNase A into PD1 Buffer to a concentration of 50 ug/ ml then store 4°C.\nToo many bacterial cells were used, reducing sample volume.\nSmearing or degrading of Plasmid DNA\nNuclease contamination\nIf used host cells have high nuclease activity (e.g., enA + strains), perform the following optional Wash Step to remove residuary nuclease.\na. After DNA Binding Step, add 400 ul of PWD Buffer into the PD Column and incubate for 2 minutes at room temperature.\nb. Centrifuge at full speed (~18,000 xg) for 30 seconds.\nc. Proceed to step 6.2.\nPlasmid DNA is not adequate for enzymatic digestions\nEluted plasmid DNA contains residual ethanol\nMake sure you have discarded the flow-through after washing with Wash Buffer (Step 6.2) and centrifuged for an additional 3 minutes (Step 7).\nDenatured Plasmid DNA migrate faster than supercoiled form during electrophoresis\nIncubation in PD2 Buffer too long\nDo not incubate the sample longer than 5 minute in PD2 Buffer"
  },
  "rnase-destroyer-sprayer": {
    "components": "APRND 001\n2 sprayer/pkg\n(200ml x 2 ea)\n& 1x sprayer\nAPRND 001-1\n1 bottle/unit\n(1000ml)\nAPRND 001-3\n2 x sprayer &\n2 x sprayer\nhead with 2\nretail boxes",
    "procedure": "Usage Note:\nWear gloves when handling RNase Destroyer Sprayer.\nContact of RNase Destroyer Sprayer to skin can cause mild irritation.\nPersonal Protection:\nHarmful by ingestion, inhalation and through skin contact. Skin light sensitive.\nStorage and Transport:\nSuitable container: polyethylene or polypropylene container packaging as recommended by manufacturer.\nStorage Incompatibility:\nAvoid storage with acids and combustible materials.",
    "specifications": "RNase Destroyer Sprayer: RNase Destroyer is ready to use and has more safty than traditional chemicals such as DEPC, a known carcinogen. RNase Destroyer, can be used to remove RNase and DNA contamination from bench, instruments, experiment tools, glass and plastic consumables. Ready to use right out of the bottle, these solutions leave no residue on work surfaces when used as directed. It is good for lab-ware and surfaces that can&rsquo;t be autoclaved.\nStorage Condition:\nStored in original containers.\nStored in a well-ventilated area.\nKeep containers securely sealed.\nProtect from light.\nProtect eyes.\nDo not mix with acids.\nStored at 2 to 30℃\nShelf Life: Valid for 6 months after sealing off.",
    "troubleshooting": ""
  },
  "plant-total-rna-mini-kit": {
    "components": "APPRK 004\n(4 preps)\nAPPRK 050\n(50 preps)\nAPPRK 100\n(100 preps)\nRB Buffer\n1.5 ml x 2\n30 ml\n60 ml\nPRB Buffer\n1.5 ml x 2\n30 ml\n60 ml\nWash Buffer 1\n1.5 ml x 2\n30 ml\n60 ml\nWash Buffer 1 * (concentrate)\n1.5 ml\n30 ml\n60 ml\nWash Buffer 2 * (concentrate)\n1.5 ml\n20 ml\n35 ml\nRNase-free Water\n0.5 ml\n6 ml\n6 ml\nFilter Column\n4 pcs\n50 pcs\n100 pcs\nRB Mini Column\n4 pcs\n50 pcs\n100 pcs\nCollection Tube\n8 pcs\n100 pcs\n200 pcs\nElution Tube\n4 pcs\n50 pcs\n100 pcs\nUser Manual\n1\n1\n1",
    "procedure": "",
    "specifications": "※ Principle : mini spin column (silica matrix)\n※ Sample size : up to 100 mg plant tissue or 1 x10 7 plant cells\n※ Operation time : 30 ~ 60 minutes\n※ Binding capacity : up to 100 ug total RNA / column\n※ Expected yield : 5 ~ 30 ug of total RNA from 100 mg of young leave\n※ Column applicability : centrifugation and vaccum\n※ Minimum elution volume : 30 ul\nImportant Notes :\n※ Make sure everything is RNase-free when handling RNA.\n※ Buffers provided in this system contain irritants. Wear gloves and lab coat when handling these buffers.\n※ Caution: &szlig;-mercaptoethanol is hazardous to human health. Perform the procedures involving RB Buffer or PRB Buffer in a chemical fume hood.\n※ Add required volume of RNase-free ethanol (96~100%) to Wash Buffer 2 when first use.\n※ Dilute RNase-free DNase 1 in reaction buffer (1M NaCl, 10 mM MnCl2, 20 mM Tris-HCl, pH 7.0 at 25&ordm;C) to final conc. = 0.5 U/ul.",
    "troubleshooting": "Problems\nPossible reasons\nSolutions\nLow or no yield of RNA\nIncorrect preparation of Wash Buffer 2.\nWash Buffer 2 is not mixed with ethanol before use.\nMake sure that the correct volumes of ethanol (96- 100 %) is added into Wash Buffer 2 when first opened.\nThe volume or the percentage of ethanol is not correct before adding into Wash Buffer 2.\nMake sure that the correct volumes of ethanol (96- 100\n%) is added into Wash Buffer 2 when first used.\nElution of genomic RNA is not efficient.\nRNase-free water is not completely absorbed by column membranes .\nAfter RNase-free water is added, stand the RB Column for 1 min before centrifugation.\nColumn is clogged\nSample is too viscous.\nReduce the sample volume.\nDegradation of eluted RNA\nSample is old.\nAlways use fresh samples."
  },
  "blood-cultured-cell-rna-mini-kit": {
    "components": "APBRK 004\n(4 preps)\nAPBRK 050\n(50 preps)\nAPBRK 100\n(100 preps)\nRL Buffer\n15 ml x 2\n120 ml\n240 ml\nRB Buffer\n1.5 ml x 2\n25 ml\n45 ml\nWash Buffer 1\n1.5 ml x 2\n30 ml\n60 ml\nWash Buffer 2 * (concentrate)\n1.5 ml\n15 ml\n35 ml\nRNase-free Water\n0.5 ml\n6 ml\n6 ml\nFilter Column\n4 pcs\n50 pcs\n100 pcs\nRB Mini Column\n4 pcs\n50 pcs\n100 pcs\nCollection Tube\n8 pcs\n100 pcs\n200 pcs\nElution Tube\n4 pcs\n50 pcs\n100 pcs\nUser Manual\n1\n1\n1",
    "procedure": "",
    "specifications": "※ Principle : mini spin column (silica matrix)\n※ Sample size : 300 ul\n※ Operation time : 30 ~ 60 minutes\n※ Binding capacity : up to 100 ug total RNA / column\n※ Expected yield : 1 ug\n※ Column applicability : centrifugation and vaccum\n※ Minimum elution volume : 40 ul\nImportant Notes :\n※ Make sure everything is RNase-free when handling RNA.\n※ Buffers provided in this system contain irritants. Wear gloves and lab coat when handling these buffers.\n※ Caution: &szlig;-mercaptoethanol is hazardous to human health. perform the procedures involving RB Buffer or PRB Buffer in a chemical fume hood.\n※ Add required volume of RNase-free ethanol (96~100%) to Wash Buffer 2 when first use.\n※ Dilute RNase-free DNase 1 in reaction buffer (1M NaCl, 10 mM MnCl2, 20 mM Tris-HCl, pH 7.0 at 25&ordm;C) to final conc. = 0.5 U/ul.",
    "troubleshooting": "Problems\nPossible reasons\nSolutions\nLow yield\nCarryover of erythrocytes.\nIncomplete removal of the supernatant will interfere with lysis and subsequent binding of RNA to the RB column, resulting in lower yield.\nToo much starting sample used may cause insufficient lysis and homogenization.\nIn Step.1 sample preparation, reduce the amount of starting sample or increase the volume of RL Buffer.\nIncorrect RNA elution step.\nEnsure that Elution Buffer was added and absorbed to the center of the PD Column matrix.\nEthanol carryover\nEnsure that centrifuge at &ge; 8000 x g (&ge;10,000 rpm) for 3 min to dry the RB column membrane in Step.6 Dry Column.\nIncorrect preparation of Wash 2 Buffer\nEnsure that the correct volume of ethanol (96 ~ 100 %) was added to Wash 2 Buffer before use.\nRNA degraded\nAge of blood.\nPlease use the fresh blood sample.\nResidual DNA\nDNA Contamination.\nPlease follow the Optional step to remove DNA.\nRNA does not perform well in downstream experiments\nSalt carryover during elution.\nEnsure that Wash 2 Buffer is at room temperature (15&ndash;25°C).\nEthanol carryover.\nPerform the step. 6 in the protocol (centrifuge at &ge; 8000 x g or &ge; 10,000 rpm) to remove all traces of ethanol before eluting."
  },
  "tissue-total-rna-mini-kit": {
    "components": "APTRK 004\n(4 preps)\nAPTRK 050\n(50 preps)\nAPTRK 100\n(100 preps)\nRB Buffer\n1.5 ml x 2\n25 ml\n45 ml\nWash Buffer 1\n1.5 ml x 2\n30 ml\n60 ml\nWash Buffer 2 * (concentrate)\n1.5 ml\n15 ml\n35 ml\nRNase-free Water\n0.5 ml\n6 ml\n6 ml\nFilter Column\n4 pcs\n50 pcs\n100 pcs\nRB Mini Column\n4 pcs\n50 pcs\n100 pcs\nCollection Tube\n8 pcs\n100 pcs\n200 pcs\nElution Tube\n4 pcs\n50 pcs\n100 pcs\nUser Manual\n1\n1\n1",
    "procedure": "",
    "specifications": "※ Principle : mini spin column (silica matrix)\n※ Sample size : Please see Table. Sample amount and Yield.\n※ Operation time : 30 ~ 60 minutes\n※ Binding capacity : up to 100 ug total RNA/ column\n※ Expected yield : Please see Table. Sample amount and Yield.\n※ Column applicability : centrifugation and vaccum\n※ Minimum elution volume : 40 ul\nImportant Notes :\n※ Make sure everything is RNase-free when handling RNA.\n※ Buffers provided in this system contain irritants. Wear gloves and lab coat when handling these buffers.\n※ Caution: &szlig;-mercaptoethanol is hazardous to human health. perform the procedures involving RB Buffer or PRB Buffer in a chemical fume hood.\n※ Add required volume of RNase-free ethanol (96~100%) to Wash Buffer 2 when first use.\n※ Dilute RNase-free DNase 1 in reaction buffer (1M NaCl, 10 mM MnCl2, 20 mM Tris-HCl, pH 7.0 at 25&ordm;C) to final conc. = 0.5 U/ul.",
    "troubleshooting": "Problems\nPossible reasons\nSolutions\nLow yield\nToo much starting sample used may cause insufficient lysis and homogenization.\nIn Step.1 sample preparation, reduce the amount of starting sample or increase the volume of RB Buffer.\nIncorrect RNA elution step .\nEnsure that Elution Buffer was added and absorbed to the center of the RB Column matrix.\nEthanol carryover .\nEnsure that centrifuge at &ge; 8000 x g (&ge; 10,000 rpm) for 3 min to dry the RB column membrane in Step.6 Dry Column.\nIncorrect preparation of Wash 2 Buffer .\nEnsure that the correct volume of ethanol (96 ~ 100 %) was added to Wash 2 Buffer before use.\nRNA degraded\nAge of Sample.\nPlease use the fresh sample.\nResidual DNA\nDNA Contamination.\nPlease follow the Optional step to remove DNA.\nRNA does not perform well in downstream experiments\nSalt carryover during elution.\nEnsure that Wash 2 Buffer is at room temperature (15&ndash;25°C).\nEthanol carryover.\nPerform the step.6 in the protocol(centrifuge at &ge; 8000 x g or &ge; 10,000 rpm) to remove all traces of ethanol before eluting."
  },
  "rna-stabilization-solution": {
    "components": "APRSS 001\nComponents\n100 ml",
    "procedure": "",
    "specifications": "RNA stabilization solution\nObtaining high quality, intact RNA is the first and often the most critical step in performing gene expression analysis. Typically, in order to isolate high quality RNA, the tissue has to be processed immediately after harvest. RNA Solution makes it possible for researchers to postpone RNA isolation for days, weeks, or even months after tissue collection without sacrificing RNA integrity. All we need to do is to add 10 times volume of RNA Solution into the tube containing the freshly collected tissue (1 ml RNA Solution to 100 mg tissue) and store the tube at &ndash;20℃ until use. In addition for RNA stabilization, RNA Solution can be easily integrated into a modified single-step RNA isolation method. This modified single-step method isolates undegraded RNA from tissues or cells in hours and can be used to process a large number of samples.",
    "troubleshooting": ""
  },
  "tri-rna-reagent": {
    "components": "APTRR 001\nComponents\n100 ml",
    "procedure": "Add 1 ml of Tri-RNA Reagent to 100 mg tissue (or precipitated blood RNA viruses from up to 10 ml of blood or 106 cultured cells, or 10 cm2 of culture plate)\nHomogenize tissue samples in Tri-RNA Reagent using a glass-Teflon or Polytron homogenizer (cultured cells can be lysed by repetitive pipetting; concentrated blood RNA viruses can be lysed by vigorous vortexing).\nLeave the homogenates for 5 minutes at room temperature.\nAdd 0.2 ml of chloroform (not provided) and mix vigorously.\nCentrifuge at 12,000 rpm for 3 minutes to separate the phases, RNA is in the clear upper aqueous phase.\nTransfer the RNA phase to a clean tube.\nRNA is precipitated by adding 1x volume of isopropanol, vortex, leave at room temperature for 10 minutes, and then centrifuge at 12,000 rpm for 15 minutes.\nRemove the supernatant.\nWash the RNA pellet with 0.5 ml ice cold 70% ethanol, centrifuge at 12,000 rpm for 1 minute, and carefully remove the supernatant.\nA brief spin to make sure the RNA pellet is precipitated to the designated side wall of the tube and then carefully remove any residue supernatant without touching the RNA pellet.\nResuspend the RNA in a small volume of TE, pH8.0.",
    "specifications": "Single-step for the isolation of total RNA from tissues, cells, bacteria, plants, yeasts and biological fluids\nThe entire procedure for total RNA isolation is less than 1 hour.\nhe purified RNA can be applied in : RT-PCR, Northern hybridlation, RNase protection, Poly-A+RNA selection, Differential display, and Micro-array assay.\nTri-RNA Reagent\nTri-RNA Reagent is a reagent from the improved phenol and guanidine isothiocyanate (GSN) method for the single-step RNA isolation.During sample homogenization or lysis, Tri-RNA reagent maintains the integrity of the RNA, while disrupting cells and dissolving cell components.The composition of Tri-RNA reagent includes phenol and GSN in a mono-phase solution.A biological sample is homogenized or lysed in Tri-RNA reagent and homogenate is separated into the aqueous and organic phases by chloroform addition, vortexing, and centrifugation.RNA remains exclusively in the aqueous phase (clear upper phase), DNA in the interphase, and proteins remain in the organic phase (red color).RNA is precipitated from aqueous phase by addition of isopropanol and solubilized/concentrated. If needed, DNA and proteins can be sequentially precipitated from the interphase and organic phase with ethanol and isopropanol respectively, and solubilized/concentrated.",
    "troubleshooting": ""
  },
  "straight-pcr-lysis-reagent-cell": {
    "components": "APDIC 100\nComponents\n100 ml\nApplications\n100 ml Cultured Cell",
    "procedure": "1. Lyse Cultured Cells in Straight PCR Lysis Reagent.\n2. Incubate for 45 min at 85°C.\n3. PCR genotyping with 1 ul lysates.",
    "specifications": "★ Time saving : Less hands-on time. Crude tail lysates for PCR.\n★ Money saving : Cost-effective reactions.\n★ Safe : No organic reagents.\n★ Environmental : Less waste (organic reagents, tubes, tips, etc...)\n★ Reliable and efficient : Virtually 100% success rate with high yields.",
    "troubleshooting": "1. Complete lysis ： Big tissue clumps should not be observed after digestion. It is recommended to vigorously shake the bottle (containing microfuge tubes) for 2-3 sec anytime, once or twice, after tissues begin to partially dissolve. This will physically disperse partially digested tissues and reposition microfuge tube, in which tails are separated from lysis reagents, thereby facilitating overall lysis efficiency .\n2. Proteinase K inactivation ： Inactivation of proteinase K by incubating samples at 85°C-86°C for 45-50 min is critical to protect Taq polymerase from proteinase K.\n3. Tissue size ： The size of tails should be 0.5 cm or slightly smaller. Use a minimal volume (0.5-1 ul for 50 ul PCR reaction) of crude lysates for PCR amplification. Too much Straight PCR Lysis Reagent inhibit PCR efficiency.\n4. Small tubes and evaporation ： To minimize evaporation, use a 0.75 ml tube when the reagent volume is less than 100 ul.\n5. Small samples and dilution ： If the required Straight PCR Lysis Reagent volume is less than 50 ul, dilute the reagent by up to 2-fold with water, while maintaining the same concentration of proteinase K. If the Straight PCR Lysis Reagent is '2-fold' diluted, apply '2-fold' more crude lysates for PCR reaction.\n6. PCR machine ： PCR machines are occasionally a source of technical problems."
  },
  "straight-pcr-lysis-reagent-ear": {
    "components": "APDIE 100\nComponents\n100 ml\nApplications\n1000 ear pieces (100ml)",
    "procedure": "1. Lyse Ear pieces in Straight PCR Lysis Reagent.\n2. Incubate for 45 min at 85°C.\n3. PCR genotyping with 1 ul lysates.",
    "specifications": "★ Time saving : Less hands-on time. Crude tail lysates for PCR.\n★ Money saving : Cost-effective reactions.\n★ Safe : No organic reagents.\n★ Environmental : Less waste (organic reagents, tubes, tips, etc...)\n★ Reliable and efficient : Virtually 100% success rate with high yields.",
    "troubleshooting": "1. Complete lysis ： Big tissue clumps should not be observed after digestion. It is recommended to vigorously shake the bottle (containing microfuge tubes) for 2-3 sec anytime, once or twice, after tissues begin to partially dissolve. This will physically disperse partially digested tissues and reposition microfuge tube, in which tails are separated from lysis reagents, thereby facilitating overall lysis efficiency .\n2. Proteinase K inactivation ： Inactivation of proteinase K by incubating samples at 85°C-86°C for 45-50 min is critical to protect Taq polymerase from proteinase K.\n3. Tissue size ： The size of tails should be 0.5 cm or slightly smaller. Use a minimal volume (0.5-1 ul for 50 ul PCR reaction) of crude lysates for PCR amplification. Too much Straight PCR Lysis Reagent inhibit PCR efficiency.\n4. Small tubes and evaporation ： To minimize evaporation, use a 0.75 ml tube when the reagent volume is less than 100 ul.\n5. Small samples and dilution ： If the required Straight PCR Lysis Reagent volume is less than 50 ul, dilute the reagent by up to 2-fold with water, while maintaining the same concentration of proteinase K. If the Straight PCR Lysis Reagent is '2-fold' diluted, apply '2-fold' more crude lysates for PCR reaction.\n6. PCR machine ： PCR machines are occasionally a source of technical problems."
  },
  "straight-pcr-lysis-reagent-mouse-tail": {
    "components": "APDIT 100\nComponents\n100 ml\nApplications\nUp to 500 tails (100ml)",
    "procedure": "1. Lyse tails in Straight PCR Lysis Reagent.\n2. Incubate for 45 min at 85°C.\n3. PCR genotyping with 1 ul lysates.",
    "specifications": "★ Time saving : Less hands-on time. Crude tail lysates for PCR.\n★ Money saving : Cost-effective reactions.\n★ Safe : No organic reagents.\n★ Environmental : Less waste (organic reagents, tubes, tips, etc...)\n★ Reliable and efficient : Virtually 100% success rate with high yields.",
    "troubleshooting": "1. Complete lysis ： Big tissue clumps should not be observed after digestion. It is recommended to vigorously shake the bottle (containing microfuge tubes) for 2-3 sec anytime, once or twice, after tissues begin to partially dissolve. This will physically disperse partially digested tissues and reposition microfuge tube, in which tails are separated from lysis reagents, thereby facilitating overall lysis efficiency .\n2. Proteinase K inactivation ： Inactivation of proteinase K by incubating samples at 85°C-86°C for 45-50 min is critical to protect Taq polymerase from proteinase K.\n3. Tissue size ： The size of tails should be 0.5 cm or slightly smaller. Use a minimal volume (0.5-1 ul for 50 ul PCR reaction) of crude lysates for PCR amplification. Too much Straight PCR Lysis Reagent inhibit PCR efficiency.\n4. Small tubes and evaporation ： To minimize evaporation, use a 0.75 ml tube when the reagent volume is less than 100 ul.\n5. Small samples and dilution ： If the required Straight PCR Lysis Reagent volume is less than 50 ul, dilute the reagent by up to 2-fold with water, while maintaining the same concentration of proteinase K. If the Straight PCR Lysis Reagent is '2-fold' diluted, apply '2-fold' more crude lysates for PCR reaction.\n6. PCR machine ： PCR machines are occasionally a source of technical problems."
  },
  "straight-pcr-lysis-reagent-yolk-sac": {
    "components": "APDIY 100\nComponents\n100 ml\nApplications\n100 ml yolk sacs",
    "procedure": "1. Lyse Yolk Sac in Straight PCR Lysis Reagent.\n2. Incubate for 45 min at 85°C.\n3. PCR genotyping with 1 ul lysates.",
    "specifications": "★ Time saving : Less hands-on time. Crude tail lysates for PCR.\n★ Money saving : Cost-effective reactions.\n★ Safe : No organic reagents.\n★ Environmental : Less waste (organic reagents, tubes, tips, etc...)\n★ Reliable and efficient : Virtually 100% success rate with high yields.",
    "troubleshooting": "1. Complete lysis ： Big tissue clumps should not be observed after digestion. It is recommended to vigorously shake the bottle (containing microfuge tubes) for 2-3 sec anytime, once or twice, after tissues begin to partially dissolve. This will physically disperse partially digested tissues and reposition microfuge tube, in which tails are separated from lysis reagents, thereby facilitating overall lysis efficiency .\n2. Proteinase K inactivation ： Inactivation of proteinase K by incubating samples at 85°C-86°C for 45-50 min is critical to protect Taq polymerase from proteinase K.\n3. Tissue size ： The size of tails should be 0.5 cm or slightly smaller. Use a minimal volume (0.5-1 ul for 50 ul PCR reaction) of crude lysates for PCR amplification. Too much Straight PCR Lysis Reagent inhibit PCR efficiency.\n4. Small tubes and evaporation ： To minimize evaporation, use a 0.75 ml tube when the reagent volume is less than 100 ul.\n5. Small samples and dilution ： If the required Straight PCR Lysis Reagent volume is less than 50 ul, dilute the reagent by up to 2-fold with water, while maintaining the same concentration of proteinase K. If the Straight PCR Lysis Reagent is '2-fold' diluted, apply '2-fold' more crude lysates for PCR reaction.\n6. PCR machine ： PCR machines are occasionally a source of technical problems."
  },
  "viral-nucleic-acid-extraction-kit": {
    "components": "APVNK 004\n(4 preps)\nAPVNK 050\n(50 preps)\nAPVNK 100\n(100 preps)\nAPVNK 200\n(200 preps)\nVNE Buffer\n1.8 ml x 2\n35 ml\n70 ml\n140 ml\nWash Buffer 1 * (concentrate)\n0.9 ml x 2\n22 ml\n44 ml\n88 ml\nWash Buffer 2 * (concentrate)\n1.5 ml\n20 ml\n40 ml\n40 ml x 2\nRNase-free Water\n0.5 ml\n6 ml\n6 ml x 2\n20 ml\nCarrier RNA\n0.05 mg\n0.4 mg\n0.8 mg\n1.6 mg\nVNE Column\n4 pcs\n50 pcs\n100 pcs\n200 pcs\nCollection Tube\n8 pcs\n100 pcs\n200 pcs\n400 pcs\nElution Tube\n4 pcs\n50 pcs\n100 pcs\n200 pcs\nUser Manual\n1\n1\n1\n1",
    "procedure": "",
    "specifications": "Principle : spin column (silica membrane)\nSample size : 150 ul cell-free fluid such as serum, plasma, body fluid and cell cultured supernatant\nFragment size : 100 bp ~30 kb\nRecovery rate : 80 ~ 90 %\nBinding capacity : 30 ug\nElution volume : 40 ~ 50 ul\nOperation time : 20 min\nImportant Notes:\nMake sure everything is RNase-free when handling this system.\nBuffers provided in this system contain irritants. Wear gloves and lab coat when handling these buffers.\nAdd 1 ml of VNE Buffer to the tube of lyophilized Carrier RNA, mix well by vortexing and transfer the mixture to the VNE Buffer when first open. Store the Carrier RNA added VNE Buffer at 4 °C.\nAdd required ethanol (96-100%) to Wash Buffer 1 and Wash Buffer 2 before use.\nPreheat RNase-free water to 70˚C for elution step.",
    "troubleshooting": "Problems\nPossible reasons\nSolutions\nLow or no yield of genomic DNA\nIncorrect preparation of Wash Buffer 1 or Wash Buffer 2\nWash Buffer 1 and Wash Buffer 2 is not mixed with ethanol before use\nMake sure that the correct volumes of ethanol (96- 100 %) is added into Wash Buffer 1 and Wash Buffer 2 when first open. Repeat the extraction procedure with a new sample.\nThe volume or the percentage of ethanol is not correct before adding into Wash Buffer 1 and Wash Buffer 2\nMake sure that the correct volumes of ethanol (96- 100\n%) is added into Wash Buffer 1 and Wash Buffer 2 when first use. Repeat the extraction procedure with a new sample.\nElution of genomic DNA is not efficient\nRNase-free water not completely absorbed by column membrane\nAfter RNase-free water is added, stand the VNE Column for 2 min before centrifugation.\nColumn is clogged\nSample is too viscous\nReduce the sample volume.\nDegradation of eluted DNA\nSample is old\nAlways use fresh or well-stored sample viral nucleic acid extraction."
  },
  "cd63-exosome-elisa-kit": {
    "components": "Exosome Protein Standard (Blue cap)\n30 μL\nCoating Buffer\n20 ml\nBlocking Buffer\n40 ml\nCD63 Probe (Green cap)\n10 μL\nHRP-conjugated Secondary Probe (Red cap)\n10 μL\nProbe Dilution Buffer\n40 mL\nWash Buffer (10X)\n50 mL\nSubstrate Buffer\n12 mL\nStop Buffer\n12 mL\n96 Well Plate\n1 plate\n(16-well x 6 strips)",
    "procedure": "Before starting\nFresh dilute CD63 Probe (2500X) to 2500-folds in Probe Dilution Buffer before step # 6. e.g. For 1 plate, add 4 uL of CD63 Probe (2500X) into 10 mL of Assay Buffer. Mix by inverting the tube.\nFresh dilute HRP-conjugated Secondary Probe (2500X) to 2500-folds in Probe Dilution Buffer before step # 8. e.g. For 1 plate, add 4 uL of HRP-conjugated Secondary Probe (2500X) into 10 mL of Assay Buffer. Mix by inverting the tube.\nELASA assay\nAdd 100 ul of freshly prepared Standard Dilutions (see protocol above) and exosome samples to the appropriate well of the 96 Well Plate .\nSeal the plate with plate seal and incubate it at 4°C overnight.\nRemove the plate seal, discard all the reaction solution. Wash each well with 350 uL of 1X Wash buffer , and then discard the buffer carefully. Repeat this step for 3 times. After the final wash, empty the wells and firmly tap the plate on a lint free paper towel to remove any remaining wash buffer. * Note: Do not use suction/aspirate to directly suck the bottom of the well.\nAdd 300 uL of Blocking Buffer to each well. Seal the plate with plate seal and incubate it at room temperature for 1 h on the Microtiter plate shaker (100 rpm).\nRemove the plate seal, discard all the reaction solution. Wash each well with 350 uL of 1X Wash buffer , and then discard the buffer carefully. Repeat this step for 3 times. After the final wash, empty the wells and firmly tap the plate on a lint free paper towel to remove any remaining wash buffer. * Note: Do not use suction/aspirate to directly suck the bottom of the well.\nAdd 100 uL of dilute CD63 Probe (1X) to each well. Seal the plate with plate seal and incubate it at room temperature for 1 h on the Microtiter plate shaker (100 rpm).\nRemove the plate seal, discard all the reaction solution. Wash each well with 350 uL of 1X Wash buffer , and then discard the buffer carefully. Repeat this step for 3 times. After the final wash, empty the wells and firmly tap the plate on a lint free paper towel to remove any remaining wash buffer. * Note: Do not use suction/aspirate to directly suck the bottom of the well.\nAdd 100 uL of dilute HRP-conjugated Secondary Probe (1X) to each well. Seal the plate with plate seal and incubate it at room temperature for 1 h on the Microtiter plate shaker (100 rpm).\nWarm the Substrate Buffer to room temperature.\nRemove the plate seal, discard all the reaction solution. Wash each well with 350 uL of 1X Wash buffer , and then discard the buffer carefully. Repeat this step for 3 times. After the final wash, empty the wells and firmly tap the plate on a lint free paper towel to remove any remaining wash buffer. * Note: Do not use suction/aspirate to directly suck the bottom of the well.\nAdd 100 uL of Substrate Buffer to each well, and then incubate the plate in dark at room temperature for 15 minutes. Add 100 uL of Stop Buffer to each well and read immediately. * Note: The initial color of a positive sample is blue and the color changes to yellow when Stop Buffer is added.\nMeasure the absorbance with a microplate reader at 450 nm.\nPlot the standard solutions data as mean OD 450.\nCalculate each sample concentration. * Note: The ideal incubation time varies depending on laboratory conditions and/or the instrument utilized. It's highly recommended to run a sample set of standards beforehand to fine-tune the assay before analyzing sensitive samples. This preliminary step assists in identifying the optimal experimental conditions for your study.",
    "specifications": "In this Indirect ELASA assay, exosomes are captured intact on the high protein binding microtiter plate. Then, CD63 probe is added, which binds to CD63 on the captured exosomes. Following wash steps, HRP-conjugated Secondary Probe and TMB solution is added, leading to a color change proportional to the amount of exosomes. The colorimetric signal is quantified using a microplate reader, providing a sensitive and specific method for exosome quantification.",
    "troubleshooting": "Problem\nPossible Cause\nSolution\nWeak or no signal\nThe protein amount is added into\nwell insufficiently\nEnsure extract contains enough amount\nof proteins\nIncubation time and temperature is incorrect\nEnsure the incubation time and temperature described in the protocol\nare correctly followed\nInadequate reagent volumes or\nimproper dilution\nCheck pipettes and ensure correct\npreparation\nHigh background\nContaminated wash buffer\nMake fresh wash buffer\nInsufficient washing\nEnsure the number of washes described\nin the protocol are correctly followed\nAnomalous signal\nReagents not fresh or not at the\ncorrect pH\nUse freshly prepared solutions and\nreagents\nReagent added in incorrect order,\nor incorrectly prepared\nReview protocol and check the\nconcentration of reagents\nBubbles in the wells\nMake sure you remove air bubbles\nwithin wells\nUneven temperature around work\nsurface\nAvoid incubating plate in areas where\nenvironmental conditions vary\nFilth at the bottom of 96 well\nplate\nMake sure the surface is clean before\nadding the reagents into wells\nThe well or plate abnormality in the experimental process\nEnsure to dry the wells thoroughly before adding the substrate. Do not let the wells dry out during incubation.\nRemove any residual liquid and\nfingerprints from the bottom of the plate to avoid affecting OD values.\nLow OD values\nIncorrect microplate reader settings\nCheck wavelength and filter settings, preheat reader.\nCheck wavelength and filter settings,\npreheat reader.\nLow sensitivity\nImproper storage of the ELISA kit\nStore reagents as per instructions\nNo stop solution added before\nreading\nAdd stop solution before reading OD"
  },
  "cd9-exosome-elisa-kit": {
    "components": "Exosome Protein Standard (Blue cap)\n30 μL\nCoating Buffer\n20 ml\nBlocking Buffer\n40 ml\nCD9 Probe (Green cap)\n10 μL\nHRP-conjugated Secondary Probe (Red cap)\n10 μL\nProbe Dilution Buffer\n40 mL\nWash Buffer (10X)\n50 mL\nSubstrate Buffer\n12 mL\nStop Buffer\n12 mL\n96 Well Plate\n1 plate\n(16-well x 6 strips)",
    "procedure": "Before starting\nFresh dilute CD9 Probe (2500X) to 2500-folds in Probe Dilution Buffer before step # 6. e.g. For 1 plate, add 4 uL of CD9 Probe (2500X) into 10 mL of Assay Buffer. Mix by inverting the tube.\nFresh dilute HRP-conjugated Secondary Probe (2500X) to 2500-folds in Probe Dilution Buffer before step # 8. e.g. For 1 plate, add 4 uL of HRP-conjugated Secondary Probe (2500X) into 10 mL of Assay Buffer. Mix by inverting the tube.\nELASA assay\nAdd 100 ul of freshly prepared Standard Dilutions (see protocol above) and exosome samples to the appropriate well of the 96 Well Plate .\nSeal the plate with plate seal and incubate it at 4°C overnight.\nRemove the plate seal, discard all the reaction solution. Wash each well with 350 uL of 1X Wash buffer , and then discard the buffer carefully. Repeat this step for 3 times. After the final wash, empty the wells and firmly tap the plate on a lint free paper towel to remove any remaining wash buffer. * Note: Do not use suction/aspirate to directly suck the bottom of the well.\nAdd 300 uL of Blocking Buffer to each well. Seal the plate with plate seal and incubate it at room temperature for 1 h on the Microtiter plate shaker (100 rpm).\nRemove the plate seal, discard all the reaction solution. Wash each well with 350 uL of 1X Wash buffer , and then discard the buffer carefully. Repeat this step for 3 times. After the final wash, empty the wells and firmly tap the plate on a lint free paper towel to remove any remaining wash buffer. * Note: Do not use suction/aspirate to directly suck the bottom of the well.\nAdd 100 uL of dilute CD9 Probe (1X) to each well. Seal the plate with plate seal and incubate it at room temperature for 1 h on the Microtiter plate shaker (100 rpm).\nRemove the plate seal, discard all the reaction solution. Wash each well with 350 uL of 1X Wash buffer , and then discard the buffer carefully. Repeat this step for 3 times. After the final wash, empty the wells and firmly tap the plate on a lint free paper towel to remove any remaining wash buffer. * Note: Do not use suction/aspirate to directly suck the bottom of the well.\nAdd 100 uL of dilute HRP-conjugated Secondary Probe (1X) to each well. Seal the plate with plate seal and incubate it at room temperature for 1 h on the Microtiter plate shaker (100 rpm).\nWarm the Substrate Buffer to room temperature.\nRemove the plate seal, discard all the reaction solution. Wash each well with 350 uL of 1X Wash buffer , and then discard the buffer carefully. Repeat this step for 3 times. After the final wash, empty the wells and firmly tap the plate on a lint free paper towel to remove any remaining wash buffer. * Note: Do not use suction/aspirate to directly suck the bottom of the well.\nAdd 100 uL of Substrate Buffer to each well, and then incubate the plate in dark at room temperature for 15 minutes. Add 100 uL of Stop Buffer to each well and read immediately. * Note: The initial color of a positive sample is blue and the color changes to yellow when Stop Buffer is added.\nMeasure the absorbance with a microplate reader at 450 nm.\nPlot the standard solutions data as mean OD 450.\nCalculate each sample concentration. * Note: The ideal incubation time varies depending on laboratory conditions and/or the instrument utilized. It's highly recommended to run a sample set of standards beforehand to fine-tune the assay before analyzing sensitive samples. This preliminary step assists in identifying the optimal experimental conditions for your study.",
    "specifications": "In this Indirect ELASA assay, exosomes are captured intact on the high protein binding microtiter plate. Then, CD9 probe is added, which binds to CD9 on the captured exosomes. Following wash steps, HRP-conjugated Secondary Probe and TMB solution is added, leading to a color change proportional to the amount of exosomes. The colorimetric signal is quantified using a microplate reader, providing a sensitive and specific method for exosome quantification.",
    "troubleshooting": "Problem\nPossible Cause\nSolution\nWeak or no signal\nThe protein amount is added into\nwell insufficiently\nEnsure extract contains enough amount\nof proteins\nIncubation time and temperature is incorrect\nEnsure the incubation time and temperature described in the protocol\nare correctly followed\nInadequate reagent volumes or\nimproper dilution\nCheck pipettes and ensure correct\npreparation\nHigh background\nContaminated wash buffer\nMake fresh wash buffer\nInsufficient washing\nEnsure the number of washes described\nin the protocol are correctly followed\nAnomalous signal\nReagents not fresh or not at the\ncorrect pH\nUse freshly prepared solutions and\nreagents\nReagent added in incorrect order,\nor incorrectly prepared\nReview protocol and check the\nconcentration of reagents\nBubbles in the wells\nMake sure you remove air bubbles\nwithin wells\nUneven temperature around work\nsurface\nAvoid incubating plate in areas where\nenvironmental conditions vary\nFilth at the bottom of 96 well\nplate\nMake sure the surface is clean before\nadding the reagents into wells\nThe well or plate abnormality in the experimental process\nEnsure to dry the wells thoroughly before adding the substrate. Do not let the wells dry out during incubation.\nRemove any residual liquid and\nfingerprints from the bottom of the plate to avoid affecting OD values.\nLow OD values\nIncorrect microplate reader settings\nCheck wavelength and filter settings, preheat reader.\nCheck wavelength and filter settings,\npreheat reader.\nLow sensitivity\nImproper storage of the ELISA kit\nStore reagents as per instructions\nNo stop solution added before\nreading\nAdd stop solution before reading OD"
  }
};
